mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -413,7 +413,8 @@ export class ExperimentView extends Component {
                     <input
                       className="ExperimentView-searchInput"
                       type="text"
-                      placeholder={'metrics.rmse < 1 and params.model = "tree"'}
+                      placeholder={'metrics.rmse < 1 and params.model = "tree" and ' +
+                                   'tags."mlflow.source.name" = "local"'}
```

Comment:
Did you intend for this to be "mlflow.source.type" = "LOCAL"?

[System Tags](https://mlflow.org/docs/latest/tracking.html#system-tags) in MLflow docs